docs/docs-new/pages/product/configuration/data-sources/duckdb.mdx

```diff
@@ -0,0 +1,115 @@
+
+## Prerequisites
+
+- [A set of IAM credentials][aws-docs-access-keys] which allow access to [AWS
+  S3][aws-s3]
+- [The AWS region][aws-docs-regions]
+- [The S3 bucket][aws-s3] on AWS to read data from
+
+## Setup
+
+### Manual
+
+Add the following to a `.env` file in your Cube project:
+
+```dotenv
+CUBEJS_DB_TYPE=duckdb
+```
+
+### Cube Cloud
+
+In Cube Cloud, select **DuckDB** when creating a new deployment and fill in the
+required fields:
+
+<InfoBox>
+
+If you are not using MotherDuck, leave the **MotherDuck Token** field blank.
+
+</InfoBox>
+
+<Screenshot
+  alt="Cube Cloud DuckDB Configuration Screen"
+  src="https://ucarecdn.com/ffad02b1-a563-47e4-bbc0-0302ab046ccd/"
+/>
+
+## Environment Variables
+
+| Environment Variable                    | Description                                                                         | Possible Values                                               | Required | [Supports multiple data sources?][ref-config-multiple-ds-decorating-env] |
+| --------------------------------------- | ----------------------------------------------------------------------------------- | ------------------------------------------------------------- | :------: | :----------------------------------------------------------------------: |
+| `CUBEJS_DUCKDB_MOTHER_DUCK_TOKEN`       | The service token to use for connections to MotherDuck                              | A valid [MotherDuck service token][motherduck-docs-svc-token] |    ❌    |                                    ✅                                    |
+| `CUBEJS_DUCKDB_S3_ACCESS_KEY_ID`        | The AWS Access Key ID to use for database connections                               | A valid AWS Access Key ID                                     |    ✅    |                                    ✅                                    |
+| `CUBEJS_DUCKDB_S3_SECRET_ACCESS_KEY_ID` | The AWS Secret Access Key to use for database connections                           | A valid AWS Secret Access Key                                 |    ✅    |                                    ✅                                    |
+| `CUBEJS_DUCKDB_S3_ENDPOINT`             | The AWS S3 endpoint                                                                 | [A valid AWS S3 endpoint][aws-docs-vpc-endpoints]             |    ✅    |                                    ✅                                    |
+| `CUBEJS_DUCKDB_S3_REGION`               | The AWS region of the Cube deployment                                               | [A valid AWS region][aws-docs-regions]                        |    ✅    |                                    ✅                                    |
+| `CUBEJS_CONCURRENCY`                    | The number of concurrent connections each queue has to the database. Default is `2` | A valid number                                                |    ❌    |                                    ❌                                    |
```

Comment:
https://github.com/cube-js/cube/pull/6982

`CUBEJS_DB_DUCKDB_MEMORY_LIMIT` - 1gb/10gb/1mb - by default it's 75% of RAM